apps/metadata-editor-e2e/src/e2e/dashboard.cy.ts

```diff
@@ -314,4 +314,16 @@ describe('dashboard', () => {
       })
     })
   })
+
+  describe('when the user is not logged in', () => {
+    beforeEach(() => {
+      cy.visit('/catalog/search')
+    })
+    it('redirects to the login page', () => {
+      cy.url().should(
+        'include',
+        '/catalog.signin?redirect=http://localhost:4200/catalog/search'
```

Comment:
Out of curiosity, where does this `redirect` value come from? The referer? It's often needed to be redirected after successful login, but I don't see it written in our code.

Reply:
it's computed here : https://github.com/geonetwork/geonetwork-ui/blob/main/libs/api/repository/src/lib/gn4/auth/auth.service.ts#L13